shared/templates/create_sysctl.py

```diff
@@ -101,29 +96,32 @@ def generate(self, target, serviceinfo):
             if not sysctl_val.strip():
 
                 # open the template files and perform the conversions
-                for sysctlfile, prefix in self.get_files_var_for_id(sysctl_var_id).items():
+                for stem, ttype in self.get_files_for_id(sysctl_var_id).items():
                     self.file_from_template(
-                        sysctlfile,
+                        "template_OVAL_sysctl",
                         {
+                            "FLAGS": ttype,
                             "SYSCTLID":  sysctl_var_id,
                             "SYSCTLVAR": sysctl_var,
+                            "SYSCTLVAL": "",
```

Comment:
Now, the difference between both calls `self.file_from_template()` seems to be only  on this `SYSCTLVAL` assignment, either `""` or `sysctl_val`.
The `if` can wrap only an assignment to `SYSCTLVAL` instead of whole `file_from_template()`

Reply:
Yes, fixed.